### PR TITLE
docs(matrix): update supported providers

### PR DESCRIPTION
**Pull request by 317brian**
DOC-289
- [x] CAPI
- [ ] CDF
- [x] EXT
- [x] Managed
- [x] PUX
- [x] SPIN

### CODE REVIEW

**Review by link108 — APPROVED** [review]

**Review by aelath — APPROVED** [review]
Looks good to me, @brian-armory

File: content/en/docs/armory-platform-matrix.md [hunk 1]
```diff
@@ -195,7 +175,7 @@ The following table lists the supported authorization methods:
 | OAuth 2.0/OIDC        | All supported versions |                                                                                      |
```
Comment (jasonmcintosh) [inline content/en/docs/armory-platform-matrix.md]: I'm a TOUCH hesitant on this.  Maybe add a note "Requires provider to include groups in claims or be a supported third party integration."

File: content/en/docs/armory-platform-matrix.md [hunk 2]
```diff
@@ -360,23 +345,20 @@ Spinnaker requires an external storage provider for persisting app settings and
...
 | S3                    | All supported versions |       |
-| Oracle Object Storage | All supported versions |       |
 
-## Manifest templating
+Depending on the service, Spinnaker also uses either Redis, MySQL, or Postgres as a backing store. The following table lists the supported database and the Spinnaker service:
 
-[![Generally available](/images/ga.svg)]({{< ref "release-definitions#ga" >}}) ![OSS](/images/oss.svg) ![Armory](/images/armory.svg)
+| Database | DB version             | Armory                 | Spinnaker services                                  | Note                                                                                                                       |
+| -------- | ---------------------- | ---------------------- | --------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------- |
+| Redis    | All supported versions | All supported versions | All Spinnaker services that require a backing store | The DB versions refer to external Redis instances. By default, Spinnaker deploys Redis internally to support its services. |
```
Comment (jasonmcintosh) [inline content/en/docs/armory-platform-matrix.md]: I'd suggest a change.  
"Supported only for services that still require Redis for a feature (e.g. gate sessions).  Not supported as a core persistent storage engine." Add in the notes "Note that the internal redis is only supported for testing/poc purposes."

**Review by dogonthehorizon — APPROVED** [review]
LGTM on behalf of PUX team

File: content/en/docs/armory-platform-matrix.md (hunk 1 above)
Comment (zachsmith1) [reply content/en/docs/armory-platform-matrix.md]: Looks like Github is the only provider in gate for oauth. Should we specifically call that out?

**Comment by andrewbackes** [comment]
@brian-armory Can we ship this and if we have to revise for CDF, do a follow up PR?

**Pull request merged**